docker-compose.yml

```diff
@@ -74,13 +83,12 @@ services:
     ports:
       - "8009:8009"
     volumes:
-      - ./chatbot/app:/app    
-    networks: 
+      - ./chatbot/app:/app
+    networks:
       - copilot-network
     depends_on:
       - db
```

Comment:
chatbot depends_on fastAPI

Reply:
I don't believe the chatbot service requires the fastapi service to be started before as they don't interact at the moment.